backend/apps/accounts/views.py

```
@@ -860,59 +904,62 @@ def get(self, request):
             )
-        
+
         # Update or create profile
         profile, created = UserProfile.objects.get_or_create(user=request.user)
-        
+
         # Delete old avatar if exists
         if profile.avatar:
             try:
                 profile.avatar.delete(save=False)
             except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)